cmd/wallet-web/src/App.vue

```diff
@@ -9,7 +9,7 @@ import { computed, onBeforeMount, onMounted, onUnmounted, ref } from 'vue';
 import { useStore } from 'vuex';
 import getStartingLocale from '@/mixins/i18n/getStartingLocale.js';
 import { updateI18nLocale } from '@/plugins/i18n';
-import IconSpinner from '@/components/icons/IconSpinner.vue';
```

Comment:
Maybe we should follow the same pattern and call our icons `spinnerIcon.vue`, `smthElseIcon.vue`, etc.

Reply:
Okay, so icon can be sort of a special component. Or could `smthIconComponent` be better for consistency..?

Reply:
Hmm `-IconComponent` sounds too bulky to me...the actual meaning is that it's an icon, even though it might have some component-like features.

Reply:
I see. I've changed it to just `SpinnerIcon` and `ErrorIcon` for now, as you suggested, I think that makes sense as well